urllib3/connectionpool.py

```diff
@@ -192,6 +194,8 @@ def __init__(self, host, port=None, strict=False,
         for _ in xrange(maxsize):
             self.pool.put(None)
 
+        self.transport_security_manager = transport_security_manager or TransportSecurityManager()
```

Comment:
Hrrrm. Ok, interesting point.

I'll review the code as-is, but I might end up giving feedback that tells you to go back. ;)